tests/v2/devfiles/metadata/metadataAll.yaml

```diff
@@ -0,0 +1,15 @@
+metadata:
+  attributes: 
+    test: Apply Test
+    scope: Api
+  name: Test-MetaData1
+  version: 1.0.0
+metadata:
+  name: test-metadata2
+metadata:  
+  version: 1.0.b 
+metadata:
+  attributes: 
+    test: metadata
+    outcome: pass
+   
```

Comment:
can we have multiple metadata top levels like this? Is this expected to fail?

Reply:
Have thought about this myself, so far nothing complains about this format so I have gone with it.